workers/docker-worker/test/integration/chain_of_trust.js

```diff
@@ -76,7 +76,7 @@ suite('certificate of trust', () => {
     let chainOfTrust = await getArtifact(result, 'public/chain-of-trust.json');
     let queue = new taskcluster.Queue(helper.optionsFromCiCreds());
     let url = queue.buildUrl(queue.getArtifact, result.taskId, result.runId, 'public/chain-of-trust.json.sig');
-    let chainOfTrustSig = (await got(url, { encoding: null })).body;
+    let chainOfTrustSig = (await got(url, { respondType: 'buffer' })).body;
```

Comment:
Is this not `responseType`?

Reply:
`respondType` is definitely not correct, yet this change fixed a test.  I'll try again..